apps/frontend/src/modules/auth/slices/auth.ts

```diff
@@ -1,9 +1,10 @@
-import { getAuthenticatedUser, signIn, signUp } from "./actions.js";
+import { getAuthenticatedUser, logout, signIn, signUp } from "./actions.js";
 import { actions } from "./auth.slice.js";
 
 const allActions = {
 	...actions,
 	getAuthenticatedUser,
+	logout,
```

Comment:
just a stylistic remark but if "Log out" contains 2 words, so maybe everywhere should be logOut, like in signIn and signUp?

Reply:
Yeah I thought about it too, but almost always logout is written using a single word (you can google it). I think if we'd use "sign out", then that would be the case. It's not that hard to change it though, if we'll need to do it, I'll do it, but for now this way of naming looks better and more convenient to me.🙌